### DISCUSSION TITLE
Problem with install symfony/orm-pack

### DISCUSSION THREAD

**Discussion by karaka200** [symfony/symfony#43795] [Q&A] [answered]
When I want to install `composer require symfony/orm-pack`, both `docker-compose.override.yml` and `docker-compose.yml` files are created in the root path. This was not the case before, but it has been around since I installed and uninstalled `Docker Desktop` for Windows 10. Does this cause a problem?

**Comment by karaka200** [comment]
Does it have to do with installing Docker Desktop or not?

**Comment by lyrixx** [comment]
IMHO, this is not normal. cc @dunglas 
(I just checked, I confirm it creates `docker-compose.override.yml` and `docker-compose.yml`)

**Reply by karaka200** [reply]
This is a bug

**Comment by BahmanMehrdad** [comment, marked as answer]
Quote from @nicolas-grekas :

"that's on purpose, see [symfony/flex#821](https://github.com/symfony/flex/pull/821)

you can opt-out by setting extra.symfony.docker to false in your composer.json file"
